src/TaskLog.php

```diff
@@ -68,4 +70,24 @@ public function __get(string $key)
             return $this->{$key};
         }
     }
+
+    /**
+     * Unify output to string.
+     *
+     * @param array|bool|int|string|null $value
+     *
+     * @return string|null
+     */
+    protected function setOutput($value)
```

Comment:
This can use native return and set to private.
```suggestion
    private function setOutput($value): ?string
```